WFP-VAM/prism-frontend
# Make downloaded CSV file consistent with analysis table

This PR closes #402.
Deploy link: http://prism-402.surge.sh/

This PR fixes the csv conversion so that the full location name stay together in one column.

To test this PR:
1. Run analysis: 10-day rainfall + IDPoor, no thresholds, any date
2. At the bottom of the table results, click 'Download' and open the CSV file

You can see that now when the language is Khmerr, the csv content is in Khmerr too:
![image](https://user-images.githubusercontent.com/30374401/166487093-4a9b72e3-e77d-4a4e-b5d3-8480f8f6d1ef.png)

Same thing when English is chosen, column "local name" is removed:
![image](https://user-images.githubusercontent.com/30374401/166487266-e4d2d5d6-6467-41eb-b32e-4690ad4e3d60.png)
